### ISSUE TITLE
Testcafe typescript fails with @babel/types/lib/index.d.ts error

### ISSUE THREAD

**Issue by oreporan** [DevExpress/testcafe#5807]
### What is your Test Scenario?
Upgraded react storybook to latest, which caused updates to `@babel`
Since then, testcafe fails on compilation of typescript

### What is the Current behavior?

Running: 

```
testcafe --skip-js-errors --selector-timeout 30000 --assertion-timeout 30000 --quarantine-mode chrome ./tests/behaviour/web
```

Results in :

```
ERROR Cannot prepare tests due to an error.

Error: TypeScript compilation failed.
/Users/xxx/node_modules/@babel/types/lib/index.d.ts (1450, 50): '{' or ';' expected.
/Users/xxx/node_modules/@babel/types/lib/index.d.ts (1450, 55): ';' expected.
/Users/xxx/node_modules/@babel/types/lib/index.d.ts (1450, 58): ';' expected.
/Users/xxx/node_modules/@babel/types/lib/index.d.ts (1452, 104): '{' or ';' expected.
....
```
 
### What is your web application and your TestCafe test code?

 
<details>
<summary>Your website URL (or attach your complete example):</summary>

</details>

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
 
```
</details>

<details>
<summary>Your complete configuration file (if any):</summary>

```

```
</details>

<details>
<summary>Your complete test report:</summary>

```

```
</details>

<details>
<summary>Screenshots:</summary>

```

```
</details>

### Steps to Reproduce:

Create a repo with storybook versions:

```
 "@storybook/addon-info": "5.3.21",
        "@storybook/addon-knobs": "6.1.11",
        "@storybook/addon-notes": "5.3.21",
        "@storybook/addon-options": "5.3.21",
        "@storybook/react": "6.1.11",
        "@types/storybook__react": "4.0.2",
        "testcafe": "^1.10.0",
```

Try to run testcafe using:

```
testcafe  chrome 
```

 
### Your Environment details:
 
* testcafe version:                   1.10.0
* node.js version:                   v12.14.0
* command-line arguments:  testcafe
* browser name and version: chrome
* platform and version:          
* other:

**Comment by hanh45** [comment]
I play with this bug in 1 day.
I fixed as:

**Comment by AlexKamaev** [comment]
@oreporan
I couldn't reproduce the issue. I performed the following steps:

* created a new project using the `npm init` command;
* added all dependencies you described in the `Steps to reproduce` section;
* ran the `testcafe chrome test.ts` command


Everything worked as expected in this case.
Try to follow these steps to reproduce the issue.
If the issue persists, delete the `node_modules` folder and the `package-lock.json` file. After that, please run `npm install` and try again.
If it does not help, please share an example so we can reproduce the issue on our side.

**Comment by sawo** [comment]
My understanding is that the babel/types/lib/index.d.ts file contains too long lines around.
When I reformatted the file above, all the bugs disappeared. I do not know, how to fix it permanently.

**Comment by Aashishgtbit** [comment]
I solve this issue by install all the @tyes/babel modules seperately.

`"@types/babel__core": "^7.1.12",
        "@types/babel__generator": "^7.6.2",
        "@types/babel__template": "^7.4.0",
        "@types/babel__traverse": "^7.11.0",`

**Comment by AlexKamaev** [comment]
Thank you all for your input.
If anybody has an example on how to reproduce the issue, please share it with us.

**Comment by oreporan** [comment]
@Aashishgtbit 

That resulted in 

```
ERROR Error: Cannot find module 'babel-plugin-transform-inline-environment-variables'
```

I don't feel going down the babel rabbit hole is  a good solution

**Comment by mihanizm56** [comment]
The same error on building cra3 with typescript default configuration. Sometimes rebuild helps, but sometimes not, this works rather strange.

**Comment by mihanizm56** [comment]
looks like the bug in babel types - think you have the same issue. Am I right?
![image](https://user-images.githubusercontent.com/37120827/102678131-83bdb280-41b7-11eb-83fe-2b9c3235ddcd.png)

**Comment by castamir** [comment]
Might be related to https://github.com/babel/babel/issues/12519 and  https://github.com/babel/babel/pull/12521

**Comment by mihanizm56** [comment]
> Might be related to [babel/babel#12519](https://github.com/babel/babel/issues/12519) and [babel/babel#12521](https://github.com/babel/babel/pull/12521)

yeah - this one!

**Comment by oreporan** [comment]
so do any changes need to be done to testcafe to make this fix work?

**Comment by mihanizm56** [comment]
> so do any changes need to be done to testcafe to make this fix work?

If it contains babel types dep then we should wait for the fixed version from babel team, I suppose.
If not - no, this is not related to testcafe directly

**Comment by castamir** [comment]
Let's wait till a new version is released, then let's verify if this bug is still present in testcafe or not.

**Comment by miherlosev** [comment]
Folks
I guess this thread mentions two problems:

* first - with TestCafe ([https://github.com/DevExpress/testcafe/issues/5808](https://github.com/DevExpress/testcafe/issues/5808))
* second - with the `@babel/types` module.
I agree with @castamir - let's wait for the new TestCafe version with the #5808 fix. After it, you can recheck your scenarios and localize the problem more precisely.

**Comment by miherlosev** [comment]
The version of TestCafe with the #5808 fix is already released - `1.10.1`.
Please recheck your scenarios and tell us about your results.

**Comment by castamir** [comment]
Our issue was resolved with new version of `@babel/types`
